include/targets/cuda/FFT_Plans.h

```diff
@@ -0,0 +1,134 @@
+#pragma once
+
+#include <quda_internal.h>
+#include <quda_matrix.h>
+#include <cufft.h>
+
+using FFTPlanHandle = cufftHandle;
+#define FFT_FORWARD     CUFFT_FORWARD
+#define FFT_INVERSE     CUFFT_INVERSE
+
+#ifndef GPU_GAUGE_ALG
+
+#define CUFFT_SAFE_CALL(call)
+
+inline void ApplyFFT(FFTPlanHandle &, float2 *, float2 *, int)
+{
+  errorQuda("CPU_GAUGE_ALG is disabled so FFTs are also disabled");
```

Comment:
that should say GPU_GAUGE_ALG instead of CPU_GAUGE_ALG, right? (multiple times)

Reply:
b798f7aba